src/public/interfaces.ts

```diff
@@ -312,6 +312,16 @@ export interface Context {
    * Unique ID for the current session for use in correlating telemetry data.
    */
   appSessionId?: string;
+
+  /**
+   * Represents whether calling is allowed for the current logged in User
+   */
+  isCallingAllowed?: boolean;
+
+  /**
+   * Represents whether PSTN calling is allowed for the current logged in User
+   */
+  isPSTNCallingAllowed?: boolean
```

Comment:
You need a semicolon at the end of this line. This is why the Travis CI build is failing.

Reply:
resolved